### ISSUE TITLE
Grain template not working with Grain 0.5.4

### ISSUE THREAD

**Issue by lann** [fermyon/spin#2779]
~~This isn't terribly important right now due to https://github.com/fermyon/spin/issues/2778~~ (easy workaround for this)

Something has gone awry with the Grain template, and it isn't (just) Spin's fault:

```console
$ cat main.gr
print("content-type: text/plain")
print("")
print("Hello, World")

$ grain --version
0.5.4
$ grain compile main.gr
$ grain run main.gr.wasm
content-type: text/plain

Hello, World

$ wasmtime --version
$ wasmtime run main.gr.wasm
content-type: text/plain
Hello, World
```

The missing newline in `wasmtime run`'s output is significant here; WAGI doesn't work without it. Spin seems to have the same problem

This seems to be fixed in Grain 0.6.6, though the file format has changed a bit there so the template would need to be updated anyway.

**Comment by lann** [comment]
It looks like grain has deeper problems with componentization: https://github.com/grain-lang/grain/issues/2153

**Comment by lann** [comment]
Should have been fixed by #2789
